subjects/devops/in-back-ground/README.md

```diff
@@ -0,0 +1,55 @@
+## in-back-ground
+
+### Instructions
+
+Create a script `in-back-ground.sh` which will execute a job in the background that will do the following:
+
+- Run the command `cat` on the file "facts" which will read the contents of the file and print it to `stdout`.
+- The output of the cat command will be piped to the `grep` command, which will search for the string `"moon"` in the file.
+- If the `grep` command succeeds (if it finds the string "moon"), the echo command will run and print `"The moon fact was found!"` to the `output` file.
+- If the command finds the string "moon", it will print the matching line to the `stdout`.
+- Redirect the `stdout` to a new file `output`.
+
+You must do all these steps running only one job and using the command `nohup`!
+
+Expected output:
+
+```console
+$ ./in-back-ground.sh
+nohup: redirecting stderr to stdout
+- Australia is wider than the moon. The moon sits at 3400km in diameter, while Australia's diameter from east to west is almost 4000km.
+$ ls
+facts in-back-ground.sh output
+$ cat output
+The moon fact was found!
+./in-back-ground.sh
```

Comment:
I am confused about this line - I guess this should be a command, but could you confirm?

Reply:
yes it is a command